tangelo/algorithms/projective/quantum_imaginary_time.py

```diff
@@ -310,6 +310,7 @@ def get_resources(self):
         resources["qubit_hamiltonian_terms"] = len(self.qubit_hamiltonian.terms)
         resources["pool_size"] = len(self.pool_operators)
         resources["circuit_width"] = self.final_circuit.width
+        resources["circuit_depth"] = self.final_circuit.depth()
```

Comment:
L313 is meant to replace L314. The number of gates ("size") of the circuit is not meaningful, the depth is.
You can remove L314 and do the same in the 2 other solvers